gooddata-java/src/main/java/com/gooddata/sdk/service/account/AccountService.java

```diff
@@ -135,17 +135,25 @@ public Account getAccountById(final String id) {
 
     /**
      * Get account for given email
```

Comment:
```suggestion
     * Get account by given login.
```
FYI: Login and email are two distinct fields in the `Account` object, so let's better be precise.